.github/workflows/ci.yml

```diff
@@ -239,9 +239,7 @@ jobs:
         run: |
           sudo apt-get install --no-install-recommends -y binfmt-support qemu-user-static
           sudo docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
-          git clone https://github.com/jeking3/bdde.git
```

Comment:
It looks like we should consider moving bdde into boostorg if we're using it for some CI situations.

Reply:
I added an issue for that such that the PR can be merged: #233